angio-frontend/src/modules/user/components/AccountActivationForm.vue

```diff
@@ -0,0 +1,174 @@
+                                prepend-inner-icon="person"
+                                counter
+                                maxlength="30"
+                                outline
+                                required
+                        ></v-text-field>
+                        <v-text-field
+                                v-model="form.fullName.patronymic"
+                                v-bind:label="$t('user.component.activation.activationFormStep.fullName.patronymic.field')"
+                                type="text"
+                                name="patronymic"
+                                prepend-inner-icon="person"
+                                counter
+                                maxlength="30"
+                                outline
+                        ></v-text-field>
+                        <v-text-field
+                                v-model="form.newPassword"
+                                v-bind:label="$t('user.component.activation.activationFormStep.newPassword.field')"
+                                v-bind:rules="[v => !!v || $t('user.component.activation.activationFormStep.newPassword.validation.NotEmpty'),
+                                    v => v.length >= 8 || $t('user.component.activation.activationFormStep.newPassword.validation.MinLength'),
+                                    form.newPassword === newPasswordRepeat || $t('user.component.activation.activationFormStep.newPassword.validation.DoesNotMatch')]"
+                                type="password"
+                                name="password"
+                                prepend-inner-icon="lock"
+                                counter
+                                maxlength="120"
+                                outline
+                                required
+                        ></v-text-field>
+                        <v-text-field
+                                v-model="newPasswordRepeat"
+                                v-bind:label="$t('user.component.activation.activationFormStep.newPasswordRepeat.field')"
+                                v-bind:rules="[v => !!v || $t('user.component.activation.activationFormStep.newPassword.validation.NotEmpty'),
+                                    v => v.length >= 8 || $t('user.component.activation.activationFormStep.newPassword.validation.MinLength'),
+                                    form.newPassword === newPasswordRepeat || $t('user.component.activation.activationFormStep.newPassword.validation.DoesNotMatch')]"
+                                type="password"
+                                name="password"
+                                prepend-inner-icon="lock"
+                                counter
+                                maxlength="120"
+                                outline
+                                required
+                        ></v-text-field>
+                        <v-expand-transition>
```

Comment:
Create separate component for form messages with prop: `errorMessages: string[]`

Reply:
Done